### ISSUE TITLE
Handling events that are not configured

### ISSUE THREAD

**Issue by sagarjs** [nickuraltsev/fluent-state-machine#7]
When i try to handle events that are not configured, that is canHandle('e) returns false, It would be great if we could specify an "else" event which gets fired.
It could be used something like this:

var fsm = finity.configure()
    .initialState('start')
    .on('a').transitionTo('Pass')
    .on('_').transitionTo('Fail');

I think this would be really good feature to have.

**Comment by nickuraltsev** [comment]
Thank you for the feedback, it's appreciated! 

Does this API look good to you?

```javascript
Finity.configure()
  .initialState('state1')
    .on('event1').transitionTo('state2')
    .onAny().transitionTo('state3')  
```

**Comment by sagarjs** [comment]
looks great!

Just wondering if 'onAny' might be a little confusing to some people? Would something like 'else' or 'onElse' or 'orAnyElse' be easier to understand?

Thanks for the prompt response, would be great to see this feature!

**Comment by nickuraltsev** [comment]
Yes, I agree that `onAny` is not ideal, but I think `else` can be even more confusing in case when there are no transitions defined for a specific event, like in the example below:

```js
Finity.configure()
  .initialState('state1')
    .else().transitionTo('state2')
```

I'm going to include this feature in the next release.

**Comment by nickuraltsev** [comment]
Closed via #10
